spark/src/test/scala/io/substrait/spark/TPCDSPlan.scala

```diff
@@ -32,16 +32,16 @@ class TPCDSPlan extends TPCDSBase with SubstraitPlanTestBase {
   }
 
   // spotless:off
-  val successfulSQL: Set[String] = Set("q1", "q3", "q4", "q7", "q8",
-    "q11", "q13", "q14a", "q14b", "q15", "q16", "q18", "q19",
-    "q21", "q22", "q23a", "q23b", "q24a", "q24b", "q25", "q26", "q27", "q28", "q29",
-    "q30", "q31", "q32", "q33", "q37", "q38",
-    "q40", "q41", "q42", "q43", "q46", "q48",
+  val successfulSQL: Set[String] = Set("q1", "q3", "q4", "q5", "q7", "q8",
```

Comment:
given how this is growing, might be worth starting to list the ones that _don't_ work instead 😄

Reply:
Exactly, I was going to do this after my next PR which will add support for more numeric functions.  The number of failing tests will be relatively small then.